### PR TITLE
feat: climate lab gallery

### PR DESCRIPTION
**Pull request by jerstucki**

### CODE REVIEW

**Review by annatraussnig — CHANGES_REQUESTED** [review]
Some remarks, one bug, nothing major. Tested on staging as well.

A couple of minor remarks:

- even though they were present while writing/previewing the card, the line breaks are gone. I know Platz ist knapp, but this also impact on readability, I feel. (Also, newlines are showing weird in FF, mobile view, prints `\n` instead of doing a line return)
- gallery, mobile view: I would have liked a less subtle indication than the font style to show that the postcard text is not normal fliesstext (maybe a grey background that also includes the illu?). It took me a few instants of looking and thinking to realise what was going on. 
- also gallery, mobile view: I think making the postcard text a bit smaller or bigger to always fit the same space (-ish) would help the ui as well. Otherwise with short texts there is this big empty space under the text. And for long texts, so have to scroll so much, it doesn't really feel like you are in a postcard anymore.

### Mobile gallery, long postcard

Screen Shot 2023-01-17 at 11 53 24

### Mobile gallery, short postcard

Screen Shot 2023-01-17 at 11 55 10

File: apps/www/pages/postcard-gallery-playground.tsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
+// import PostcardGallery from '../components/Climatelab/Postcard/Gallery/PostcardGallery'
```
[the remark this answers is not in the document]
Comment (felixmichel) [reply apps/www/pages/postcard-gallery-playground.tsx:1]: dev purpose only

[the remark this answers is not in the document]
Comment (annatraussnig) [reply apps/www/pages/postcard-gallery-playground.tsx:1]: we need to delete it before merging then, right?

File: apps/www/components/Climatelab/Postcard/PostcardGenerator.js [hunk 2]
```diff
@@ -47,9 +47,32 @@ const SubmittedPostcard = (props) => {
     scrollIntoView(postcardRef.current)
   }, [])
 
+  console.log(questionnaire)
```
Comment (annatraussnig) [inline apps/www/components/Climatelab/Postcard/PostcardGenerator.js:50]: a wild log appeared…

**Review by annatraussnig — APPROVED** [review]

File: apps/www/components/Climatelab/Postcard/PostcardGenerator.js (hunk 2 above)
Comment (felixmichel) [reply apps/www/components/Climatelab/Postcard/PostcardGenerator.js:50]: shall be removed, i think i forgot it for some weird reason

**Pull request merged**